decisions/wc_offmap_decisions.txt

```diff
@@ -616,7 +724,135 @@ offmap_decisions = {
 					}
 				}
 			}
-			FROM = { hidden_effect = { set_character_flag = requested_offmap_power } }
+			FROM = {
+				hidden_effect = {
+					set_character_flag = requested_offmap_power
+					random_list = {
+						5 = {
+							modifier = {
+								factor = 4
+								check_variable = { which = void_corruption value = 77 }
+							}
+							modifier = {
+								factor = 2
+								check_variable = { which = void_corruption value = 88 }
+							}
+							modifier = {
+								factor = 2
+								check_variable = { which = void_corruption value = 99 }
+							}
+						}
+						100 = {
+							add_10_void_corruption_effect = yes
```

Comment:
It shouldn't be in `hidden_effect` at all. It has a nice tooltip telling you will get 10 Void Corruption, never hide it.